### PR TITLE
sqlcl: add version 22.2.0.172.1718

### PR DESCRIPTION
**Pull request by delassiops**
Closes #XXXX

Relates to #XXXX

- [ ] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/sqlcl.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "version": "22.2.0.172.1718",
+    "description": "Oracle SQL Developer Command Line (SQLcl)",
+    "homepage": "https://www.oracle.com/database/sqldeveloper/technologies/sqlcl/",
+    "license": "Oracle Free Use License",
+    "suggest": {
+        "JRE": "java/openjdk"
+    },
+    "url": "https://download.oracle.com/otn_software/java/sqldeveloper/sqlcl-22.2.0.172.1718.zip",
+    "hash": "eb69022227fec14578e319a398bd20958e94da77b263300ba09af8bd5aac4aa1",
+    "bin": "sqlcl\\bin\\sql.exe",
```
Comment (rashil2000) [inline bucket/sqlcl.json:11]:
```suggestion
    "extract_dir": "sqlcl",
    "bin": "bin\\sql.exe",
```

File: bucket/sqlcl.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "22.2.0.172.1718",
+    "description": "Oracle SQL Developer Command Line (SQLcl)",
+    "homepage": "https://www.oracle.com/database/sqldeveloper/technologies/sqlcl/",
+    "license": "Oracle Free Use License",
```
Comment (rashil2000) [inline bucket/sqlcl.json:5]:
```suggestion
    "license": {
        "identifier": "Oracle Free Use License",
        "url": "https://www.oracle.com/downloads/licenses/oracle-free-license.html"
    },
```

Comment (delassiops) [reply bucket/sqlcl.json:5]: i will test, there is issue with checkver

Comment (rashil2000) [reply bucket/sqlcl.json]: Checkver seems fine, but PowerShell seems to be unable to load the URL. Any ideas @issaclin32?

Comment (issaclin32) [reply bucket/sqlcl.json]: Yeah. `wget` could not load the page due to cookie storage permissions.
It is totally fine for users who open the page with a browser, but there could be potential problems in automated workflows.

How about using the snapshot from **archive.org**?
https://web.archive.org/web/20220615160155/https://www.oracle.com/downloads/licenses/oracle-free-license.html

**Comment by delassiops** [comment]
updated checkver url, please verify with checkver.ps1 script

File: bucket/sqlcl.json (hunk 2 above)
Comment (rashil2000) [reply bucket/sqlcl.json]: > How about using the snapshot from archive.org?

For checkver? The archive.org pages will not get updated, right?

**Comment by delassiops** [comment]
for checkver url i used Homebrew Mac Package manager  webpage, please verify

File: bucket/sqlcl.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+    "version": "22.2.0.172.1718",
+    "description": "Oracle SQL Developer Command Line (SQLcl)",
+    "homepage": "https://www.oracle.com/sqlcl",
```
Comment (rashil2000) [inline bucket/sqlcl.json:4]:
```suggestion
    "homepage": "https://www.oracle.com/database/sqldeveloper/technologies/sqlcl/",
```

It's better to use non-redirecting URLs

File: bucket/sqlcl.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "license": {
+        "identifier": "Oracle Free Use License",
+        "url": "https://www.oracle.com/downloads/licenses/oracle-free-license.html"
+    },
+    "suggest": {
+        "JRE": "Java 11+"
```
Comment (rashil2000) [inline bucket/sqlcl.json:10]:
```suggestion
        "JRE": "java/openjdk"
```

This needs to be an actual package name

**Comment by delassiops** [comment]
verified 👍

File: bucket/sqlcl.json (hunk 2 above)
Comment (issaclin32) [reply bucket/sqlcl.json]: Oh. I thought you were talking about the license page.

**Pull request merged**